PushSignedBinariesIntoBuild.cmd

```diff
@@ -22,24 +22,34 @@ set BinaryOutputDirectory=%BinaryOutputDirectory%\%Platform%_%Configuration%\
 set SigningDirectory=%BinaryOutputDirectory%\..\Signing
 
 :: Copy all multitargeted assemblies to their locations
-call :CopyFilesForMultitargeting Json.Schema.dll            || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Pointer.dll           || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Schema.ToDotNet.dll   || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Schema.Validation.dll || goto :ExitFailed
+call :CopyDllForMultitargeting Json.Schema.dll            || goto :ExitFailed
+call :CopyDllForMultitargeting Json.Pointer.dll           || goto :ExitFailed
+call :CopyDllForMultitargeting Json.Schema.ToDotNet.dll   || goto :ExitFailed
+call :CopyDllForMultitargeting Json.Schema.Validation.dll || goto :ExitFailed
+
+call :CopyExeForMultitargeting Json.Schema.ToDotNet.Cli   || goto :ExitFailed
```

Comment:
>CopyExeForMultitargeting [](start = 6, length = 24)

Consider: `CopySignedExecutableToBuildOutputDirectory`. "Executable" rather than "Exe" because the latter suggests a filename extension, and the file being copied isn't always a `.exe`.